public/assests/js/message.js

```diff
@@ -203,13 +203,21 @@ function updateConversation(data) {
       var messageText = $("<div></div>").addClass("message-text");
       
       // Embed YouTube video if link presents in message.
+      var result = anchorme(data[i].message.replace(/<(?:.|\n)*?>/gm, ''), { attributes:[ { name:"target", value:"_blank" } ] });
       var youtube_link = data[i].message.indexOf('https://www.youtube.com');
       if(youtube_link !== -1) {
```

Comment:
Rather repeating the same code again and again, it would be better to make a new function and cal that function on getting link similar to http://youtube.com or http://youtu.be/ else will send message. What do you say ??